source/beamline/CLS/CLSSIS3820ScalerContinuousMode.h

```diff
@@ -1,31 +1,25 @@
-#ifndef CLSSIS3820SCALERMODECONTROL_H
-#define CLSSIS3820SCALERMODECONTROL_H
+#ifndef CLSSIS3820SCALERCONTINUOUSMODE_H
+#define CLSSIS3820SCALERCONTINUOUSMODE_H
 
-#include "beamline/AMPseudoMotorControl.h"
+#include "beamline/AMEnumeratedControl.h"
 
-class CLSSIS3820ScalerModeControl : public AMPseudoMotorControl
+class CLSSIS3820Scaler;
+
+class CLSSIS3820ScalerContinuousMode : public AMEnumeratedControl
```

Comment:
There are at least two different 'mode groups' for the scaler: Continuous/SingleShot, InputMode. The purpose of renaming this class was to differentiate between the two. If you think the old one was fine, I can change it back.